API/jsi/jsi/jsi.h

```diff
@@ -267,6 +270,12 @@ class JSI_EXPORT Runtime {
   /// which returns no metrics.
   virtual Instrumentation& instrumentation();
 
+  /// Creates a Node-API environment.
+  /// \throw a \c JSINativeException if the runtime does not support Node-API.
+  /// \param napi_env the output parameter used to store the newly created
+  /// Node-API environment.
+  virtual void createNodeApiEnv(int32_t apiVersion, napi_env *);
```

Comment:
I would propose to follow the JSI style and return the `napi_env` as `void*` directly from the function:
`virtual void* createNodeApiEnv(int32_t apiVersion);`